app/2.0/docs/upgrade.md

```diff
@@ -72,48 +72,60 @@ npm update polymer-cli@next
 
 #### Create a new branch or workarea
 
-This should go without saying, but you'll want to work in a new branch or workarea so you don't
+You'll want to work in a new branch or workarea so you don't
 break the existing, 1.x version of your element or app.
 
 #### Update bower dependencies
 
-Update the Polymer version in `bower.json` to the latest RC versions.
+Follow these steps to update your bower dependencies:
 
-| Component | Version |
-|-----------|---------|
-| Polymer   | `^2.0.0-rc.3` |
-| webcomponentsjs | `^1.0.0-rc.7` |
-| web-component-tester | `^6.0.0-prerelease.6` |
-| Polymer elements | `2.0-preview` |
+1.  Remove the existing `bower_components` folder.
 
-Note that Polymer, webcomponentsjs, and web-component-tester have prerelease tags. You can use
-a range like `^2.0.0-rc.3` to get all future prerelease tags. To use prerelease elements,
-you need to refer to the `2.0-preview` branch. To get  updates to elements, you may need to remove
-your `bower_components` folder and reinstall all components.
+    ```bash
+    rm -rf bower_components
+    ```
 
-Example dependencies {.caption}
+2.  Update the Polymer version in `bower.json` to the latest RC versions.
 
-```
-  "dependencies": {
-    "app-layout": "PolymerElements/app-layout#2.0-preview",
-    "app-route": "PolymerElements/app-route#2.0-preview",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#2.0-preview",
-    "iron-icon": "PolymerElements/iron-icon#2.0-preview",
-    "iron-iconset-svg": "PolymerElements/iron-iconset-svg#2.0-preview",
-    "iron-localstorage": "PolymerElements/iron-localstorage#2.0-preview",
-    "iron-media-query": "PolymerElements/iron-media-query#2.0-preview",
-    "iron-pages": "PolymerElements/iron-pages#2.0-preview",
-    "iron-selector": "PolymerElements/iron-selector#2.0-preview",
-    "paper-icon-button": "PolymerElements/paper-icon-button#2.0-preview",
-    "polymer": "Polymer/polymer#^2.0.0-rc.3",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.7"
-  },
-  "devDependencies": {
-    "web-component-tester": "^6.0.0-prerelease.6"
-  },
-```
+    | Component | Version |
+    |-----------|---------|
+    | Polymer   | `^2.0.0-rc.3` |
+    | webcomponentsjs | `^1.0.0-rc.7` |
+    | web-component-tester | `^6.0.0-prerelease.6` |
+    | Polymer elements | `2.0-preview` |
+
+    Note that Polymer, webcomponentsjs, and web-component-tester have prerelease tags. You can use
+    a range like `^2.0.0-rc.3` to get all future prerelease tags. To use prerelease elements,
+    you need to refer to the `2.0-preview` branch. To get  updates to elements, you may need to remove
```

Comment:
"To get updates to elements, you may need to remove your `bower_components` folder and reinstall all components"

We have instructed the user to do so, so might be able to remove this statement. idk, not too important.